packages/eslint-plugin/README.md

```diff
@@ -19,11 +19,13 @@ Enable the rules you want to use within rules property of your .eslintrc:
 {
   rules: {
     "@kaizen/no-draft-imports": "error",
+    "@kaizen/no-inline-notification-imports-from-component-library": "error"
```

Comment:
For the stylelint plugin we ended up defining a simple preset that included all the recommended rules so that the consumers weren't expected to constantly be tweaking their Kaizen eslint rules every time we write a new one. (See readme: https://github.com/cultureamp/kaizen-design-system/blob/master/packages/stylelint-plugin/README.md#getting-started) 

I feel like it's probably worth us setting up something similar here so we have control over which rules are enabled and disabled by default.